analyzers/rspec/cs/S3329_c#.html

```diff
@@ -29,16 +29,13 @@ <h2>Compliant Solution</h2>
 <pre>
 public byte[] Encrypt(byte[] key, byte[] data, MemoryStream target)
 {
-    using (AesCryptoServiceProvider aes = new AesCryptoServiceProvider())
-    {
-        aes.GenerateIV();
-        ICryptoTransform encryptor = aes.CreateEncryptor(key, aes.IV);
-        using (CryptoStream cryptoStream = new CryptoStream(target, encryptor, CryptoStreamMode.Write))
-        {
-            cryptoStream.Write(data);
-        }
-        return aes.IV;
-    }
+    using AesCryptoServiceProvider aes = new AesCryptoServiceProvider();
+    var encryptor = aes.CreateEncryptor(key, aes.IV);
```

Comment:
Should we explain what's going on and why?
```suggestion
    var encryptor = aes.CreateEncryptor(key, aes.IV); // aes.IV is automatically generated to random secure value
```